Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -469,6 +469,8 @@ public static function initializeObjectManagerCompileTimeFinalize(Bootstrap $boo
         $objectManager->injectLogger($logger);
         $objectManager->initialize($packageManager->getAvailablePackages());
 
+        $cacheManager->injectReflectionService($reflectionService);
```

Comment:
A bit hackish to inject the reflection service into the CacheManager in the initialization of the ObjectManager, but that was the simplest place and right before the cache manager is made generally available. Otherwise, this would need another compiletime step that only does this.